tensorflow_addons/image/transform_ops.py

```diff
@@ -309,6 +309,53 @@ def _image_projective_transform_grad(op, grad):
     return [output, None, None]
 
 
+def extend(
+    images: TensorLike,
+    length: Union[int, Tuple[int, int]],
+) -> tf.Tensor:
+    """Extend image(s) by some length by copying pixels on border.
+
+    Args:
+      images: A tensor of shape
+        (num_images, num_rows, num_columns, num_channels) (NHWC),
+        (num_rows, num_columns, num_channels) (HWC), 
+        or (num_rows, num_columns) (HW).
+      length: One of the following
+        A scalar length to extend the image by on each side. 
+        A tuple of pairs (LH, LW) to extend the image by in each dimension.
+        A tuple of tuple pairs ((LT, LB), (LL, LR)) to extend the image by for
+        each individual side
```

Comment:
This is not implemented yet, right?

Reply:
Oops! I intended it to add it before but it slipped my mind.